packages/next/build/babel/preset.ts

```diff
@@ -59,13 +59,25 @@ module.exports = (
   options: NextBabelPresetOptions = {}
 ): BabelPreset => {
   const supportsESM = api.caller(supportsStaticESM)
+  const isServer = api.caller((caller: any) => !!caller && caller.isServer)
   const presetEnvConfig = {
     // In the test environment `modules` is often needed to be set to true, babel figures that out by itself using the `'auto'` option
     // In production/development this option is set to `false` so that webpack can handle import/export with tree-shaking
     modules: 'auto',
     exclude: ['transform-typeof-symbol'],
     ...options['preset-env'],
   }
+
+  // When transpiling for the server, target the current Node version if not explicitly specified:
+  if (
+    isServer &&
+    (!presetEnvConfig.targets || !('node' in presetEnvConfig.targets))
+  ) {
+    presetEnvConfig.targets = {
+      node: true,
```

Comment:
```suggestion
      // Targets the current process' version of Node. This requires apps be
      // built and deployed on the same version of Node.
      node: 'current',
```

Can we use `current` over `true` for clarity?

Reply:
yup, good call.